stable/datadog/values.yaml

```
@@ -360,6 +360,23 @@ clusterAgent:
   #
   # readinessProbe:
 
+  ## @param service - object - required
```

I would prefer to have this parameter optional 
```suggestion
  ## @param service - object - optional
  ##
  # serviceType: ClusterIP
```
And I think that `annotations` is not used.